util/app-config/Cargo.toml

```diff
@@ -21,7 +21,7 @@ ckb-resource = { path = "../../resource"}
 ckb-build-info = { path = "../build-info" }
 ckb-types = { path = "../types" }
 ckb-fee-estimator = { path = "../fee-estimator" }
-p2p = { version="0.3.0-alpha.4", package="tentacle", features = ["molc"] }
+p2p = { version="0.3.0", package="tentacle", features = ["molc"] }
```

Comment:
In current commit, this version is not as same as `ckb-network`,  and only `p2p::secio::PeerId` is used in `ckb-app-config`.
So I think the dependencies in `ckb-network` and `ckb-app-config` could be different.

_As @doitian said in https://github.com/nervosnetwork/ckb/pull/2199#discussion_r465498134_:
> It's better to create a config crate so that app config does not depend on services.

Thus, could you change this to a lightweight crate?